app/views/exports/hamilton_pooling_plate_pbmc.erb

```diff
@@ -37,6 +37,4 @@
     data + [samples_in_pool * 3.125]
   end
 %>
-<% rows_array.sort_by{ |a| WellHelpers.well_coordinate(a[1]) }.each do |row| %>
-<%= CSV.generate_line row, row_sep: '' %>
-<% end %>
+<%= rows_array.sort_by{ |a| WellHelpers.well_coordinate(a[1]) }.map(&:to_csv).join %>
```

Comment:
Did you test this still works? (Just a bit cautious of that `(&:)` syntax as it mysteriously doesn't work in some situations...

Reply:
Yes the unit tests confirm that the generated CSV is identical.